### PR TITLE
Change small details

### PR DESCRIPTION
**Pull request by tcrespog**
Just changing small details that I noticed while reading the docs.  Feel free to discard or modify further.

### CODE REVIEW

File: content/docs/getting-started/usage.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ To try Tower, visit [tower.nf](https://tower.nf/login) and login with GitHub or
 <br>
 
 ## Community
-For more information on installing the Community version of Tower visit [our GitHub repository](https://github.com/seqeralabs/nf-tower) and follow our [installation guide](/docs/compute-envs).
+For more information on installing the Community version of Tower visit [our GitHub repository](https://github.com/seqeralabs/nf-tower) and follow our [installation guide](/docs/compute-envs)?.
```
Comment (tcrespog) [inline content/docs/getting-started/usage.md:43]: Is the `/docs/compute-envs` URL correct as link to the installation guide?

Comment (alaincoletta) [reply content/docs/getting-started/usage.md:43]: no it's not 👍, should be this: `docs/installation/system-deployment/` 

I will submit a change. Thanks

Comment (pditommaso) [reply content/docs/getting-started/usage.md:43]: still pending this?

Comment (tcrespog) [reply content/docs/getting-started/usage.md:43]: > still pending this?

Shall I change it as part of this PR or are you going to do it @alaincoletta?

Comment (tcrespog) [reply content/docs/getting-started/usage.md]: I've just changed it myself:https://github.com/seqeralabs/nf-tower-docs/pull/22/files#diff-de40f3d5515902b327639def0333f59a6ff3c12ab1532bbbf49f80f507def87bR43 :+1:

Comment (alaincoletta) [reply content/docs/getting-started/usage.md]: Sorry I'm still getting used to the PR process. Thanks!

**Review by alaincoletta — APPROVED** [review]
typos and wrong link fixed

**Pull request merged**